src/planner/planner.cpp

```diff
@@ -187,6 +191,21 @@ std::unique_ptr<LogicalPlan> Planner::planCopy(
     return plan;
 }
 
+std::unique_ptr<LogicalPlan> Planner::planCopyTo(const Catalog& catalog,
+    const NodesStatisticsAndDeletedIDs& nodesStatistics, const RelsStatistics& relsStatistics,
+    const BoundStatement& statement) {
+    auto& copyClause = reinterpret_cast<const BoundCopy&>(statement);
+    auto regularQuery = copyClause.getRegularQuery();
+    if (regularQuery->getStatementType() != StatementType::QUERY) {
```

Comment:
I think the parser has the guarantee for this already. So you can assert here rather than check and throw